hack/pinned-bundle-images.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
```

Comment:
I would like to fashion a way to share the logic here with pin-custom-bundle-dockerfile.sh as it is largely the same. Perhaps can do that in a follow up.